server/game/cards/06-CotE/Unmask.js

```diff
@@ -0,0 +1,32 @@
+const DrawCard = require('../../drawcard.js');
+const { Players, CardTypes } = require('../../Constants');
+
+class Unmask extends DrawCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Discard a character\'s status token and set its skills to its printed value until the conflicts end. Its controller gains 2 honor.',
```

Comment:
This is way too long - remember that the reason for a title is to allow a player to distinguish between abilities in a prompt.

Reply:
Changed to "Discard a character's status token and set skills to printed value"